scripts/zones/Mamook/npcs/Toad_s_footprint1.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+-- Area: Mamook
+-- NPC: Toads Footprint
+-- !pos 5.9660 42.7575 -100.5741
+-----------------------------------
+local ID = require("scripts/zones/Mamook/IDs")
+require("scripts/globals/npc_util")
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player, npc)
+    local princeandhopper = player:getCharVar("princeandhopper")
+    if  princeandhopper == 3 then
+        player:startEvent(223)
+    elseif  princeandhopper == 4 then
+        player:messageSpecial(ID.text.DRAWS_NEAR)
+        npcUtil.popFromQM(player, npc, ID.mob.POROGGO_CASANOVA, {hide = 30})
```

Comment:
You might want this one to spawn last. I think with the recent claim changes only the last one spawned will stay claimed after about 3 seconds unless engaged.

Reply:
So just switch casa to the bottom of the list eh?

Reply:
Probably, that one looks like the main target and if any of them should have their claim stick it'd be that one.